src/ggrc/views/converters.py

```diff
@@ -24,6 +24,7 @@
 from flask import json
 from flask import render_template
 from flask import request
+from sqlalchemy import exc
```

Comment:
To make it more clean:
```suggestion
from sqlalchemy import exc as sa_exc
```